### PR TITLE
Remove getOrderId from PurchaseHistory

### PR DESCRIPTION
**Pull request by rborn**
PurchaseHistoryRecord doesn't have orderId

### CODE REVIEW

File: android/src/main/java/com/dooboolab/RNIap/RNIapModule.java [hunk 1]
```diff
@@ -319,7 +319,12 @@ public void onPurchaseHistoryResponse(BillingResult billingResult, List<Purchase
             for (PurchaseHistoryRecord purchase : purchaseHistoryRecordList) {
               WritableMap item = Arguments.createMap();
               item.putString("productId", purchase.getSku());
-              item.putString("transactionId", purchase.getOrderId());
+              
+              // item.putString("transactionId", purchase.getOrderId());
+              // PurchaseHistoryRecord doesn't have orderId, 
+              // and it seems that getOriginalJson in this case also doesn't return the orderId
+              // is there any way to obtain this?
```
Comment (hyochan) [inline android/src/main/java/com/dooboolab/RNIap/RNIapModule.java]: Not in `android`'s billing client version 2. Why not using getPurchases instead to acquire this?

**Comment by rborn** [comment]
My Java knowledge is limited, I removed this for now as it crashes otherwise 😞

**Comment by hyochan** [comment]
@rborn Yes removing should be accepted. Can I merge this now?

**Comment by rborn** [comment]
Yes please 😻

**Comment by hyochan** [comment]
@rborn This repo doesn't seem to be `recent`, therefore I'll try rebasing it now. Could you kindly give us an email to `dooboolab@gmail.com`, your email address so I can add you to co-committer for credit?

**Comment by hyochan** [comment]
Preparing for `3.1.2` release for current bugfix. Since I want to give @rborn credit, please tell me your github email address via email <dooboolab@gmail.com>.

Closed for #556

**Comment by rborn** [comment]
@hyochan thank you ❤️

**Pull request closed without merging**